tests/pyreverse/test_pyreverse_functional.py

```diff
@@ -23,17 +22,15 @@
     CLASS_DIAGRAM_TESTS,
     ids=CLASS_DIAGRAM_TEST_IDS,
 )
-def test_class_diagrams(
-    testfile: FunctionalPyreverseTestfile, tmpdir: LocalPath
-) -> None:
+def test_class_diagrams(testfile: FunctionalPyreverseTestfile, tmp_path: Path) -> None:
     input_file = testfile.source
     for output_format in testfile.options["output_formats"]:
         with pytest.raises(SystemExit) as sys_exit:
-            args = ["-o", f"{output_format}", "-d", str(tmpdir)]
+            args = ["-o", f"{output_format}", "-d", str(tmp_path)]
             args.extend(testfile.options["command_line_args"])
             args += [str(input_file)]
             Run(args)
         assert sys_exit.value.code == 0
         assert testfile.source.with_suffix(f".{output_format}").read_text(
             encoding="utf8"
-        ) == Path(tmpdir / f"classes.{output_format}").read_text(encoding="utf8")
+        ) == Path(tmp_path / f"classes.{output_format}").read_text(encoding="utf8")
```

Comment:
```suggestion
        ) == (tmp_path / f"classes.{output_format}").read_text(encoding="utf8")
```
As `tmp_path` is already a `Path` instance we can skip this here.